wskdeploy-template/minimal/src/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Should we add any guides about how to use this plugin somewhere?
How to create a basic template, how to run a test, basic architecture, and so on?
I am just asking your opinion.